services/webnn/ort/platform_functions_ort.cc

```diff
@@ -28,8 +28,9 @@ PlatformFunctions::PlatformFunctions() {
     return;
   }
 
-  // TODO: Consider checking the version of the loaded onnxruntime.dll before
-  // loading all these functions to avoid runtime crashes.
+  // TODO(https://github.com/shiyi9801/chromium/issues/56): Consider checking
```

Comment:
This issue can be closed now I think, could we remove this TODO?

Reply:
Removed.